@here/harp-utils/lib/GroupedPriorityList.ts

```diff
@@ -45,7 +45,7 @@ export type PriorityListGroupMap<T extends PriorityListElement> = Map<number, Pr
  */
 export class GroupedPriorityList<T extends PriorityListElement> {
     readonly groups: PriorityListGroupMap<T> = new Map();
-
+    private m_sortedGroups: Array<PriorityListGroup<T>> | undefined;
```

Comment:
Why do we need this? Does it matter in which order we overlay the labels?

Reply:
This is a merging issue. I introduced this to fix the disappearing point labels (HARP-6843: Labels disappear after initial load)

Reply:
not really, but I can't index the priority lists in the "groups" map, that's why I use the sorted groups array.

Reply:
OK, my bad. No merging issue.